docs/contributor/releasing.md

```diff
@@ -32,16 +32,22 @@ This release process covers the steps to release new major and minor versions fo
 
 6. Merge the PR.
 
-7. In the `telemetry-manager/{RELEASE_BRANCH}` branch, create release tags for the head commit.
+7. Make sure that the release tags will point to the HEAD commit of the `telemetry-manager/{RELEASE_BRANCH}` branch. This can be done, by rebasing the upstream branch into the local branch after the merge was successful.
```

Comment:
```suggestion
7. To make sure that the release tags point to the HEAD commit of the `telemetry-manager/{RELEASE_BRANCH}` branch, rebase the upstream branch into the local branch after the merge was successful.
```

Do we really need "after the merge was successful"? Merging is the previous step; would anyone really do step 7 before they see the merge has happened successfully?

Reply:
I did it yesterday 🥲